docker-compose.yaml

```diff
@@ -77,19 +54,17 @@ services:
       - SETTING_KEYCLOAK_REALM=quantum-serverless
       - SETTINGS_KEYCLOAK_CLIENT_SECRET=supersecret
```

Comment:
Here I think we can just remove `SETTING_KEYCLOAK_*` configuration

Reply:
resolved in https://github.com/Qiskit-Extensions/quantum-serverless/pull/1132/commits/08c62782bba3344fe358a1c31a2149472269dea9